package.json

```diff
@@ -4,17 +4,52 @@
   "scripts": {
     "build": "node ./scripts/build.js",
     "clean": "git clean -fdX .",
-    "postinstall": "node ./scripts/postinstall.js",
+    "postinstall": "lerna run prepare",
     "start": "node ./scripts/start.js",
     "test": "node ./scripts/test.js"
   },
   "devDependencies": {
+    "@babel/core": "^7.1.2",
+    "@babel/plugin-proposal-class-properties": "^7.1.0",
+    "@babel/plugin-transform-runtime": "^7.1.0",
+    "@babel/preset-env": "^7.1.0",
+    "@babel/preset-react": "^7.0.0",
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-eslint": "^10.0.1",
+    "babel-jest": "^23.6.0",
+    "babel-plugin-dev-expression": "^0.2.1",
+    "eslint": "^5.7.0",
+    "eslint-plugin-import": "^2.12.0",
+    "eslint-plugin-react": "^7.9.1",
+    "history": "^4.9.0",
     "husky": "^1.1.3",
+    "jest": "^24.5.0",
```

Comment:
I also updated Jest because of that Node 11.11/Jest `Symbol` issue. I do believe that there is also a Node fix, but I figured we might as well get Jest up to date.